src/User.php

```diff
@@ -557,15 +557,15 @@ public function getFromDBbySyncField($value)
     /**
      * Retrieve a user from the database using it's dn.
      *
-     * @since 0.84
-     *
      * @param string $user_dn dn of the user
      *
      * @return boolean
      */
     public function getFromDBbyDn($user_dn)
     {
-        return $this->getFromDBByCrit(['user_dn' => $user_dn]);
+        return $this->getFromDBByCrit([
```

Comment:
maybe add a comment why getting the user by it's hash here.
The future maintener will thnak you